sync/class.jetpack-sync-module-plugins.php

```diff
@@ -30,6 +31,25 @@ public function init_before_send() {
 	}
 
 	public function check_upgrader( $upgrader, $details) {
+		/*
+		 * Handle plugin update errors
+		 */
+		if (
+			'Plugin_Upgrader' == get_class( $upgrader ) &&
+			'WP_Ajax_Upgrader_Skin' == get_class( $upgrader->skin ) &&
+			isset( $details['type'] ) &&
+			'plugin' == $details['type']
+		) {
+			$errors = $upgrader->skin->get_errors();
+			if (
+				'WP_Error' == get_class( $errors ) &&
+			     ! empty ( $errors )
+			) {
+				do_action( 'jetpack_plugin_update_failed', $details, $errors->errors, $errors->error_data );
```

Comment:
Could you add a docblock for this hook, so it can be picked up by the parser?

Thank you!